src/App.jsx

```diff
@@ -1,56 +1,94 @@
 import "./App.css"
+import React, { useState } from 'react';
 
 function App() {
+  const [firstPanelNum, updateNum1] = useState(0);
+  const [secondPanelNum, updateNum2] = useState(0);
+  const [operator, setOperator] = useState('+');
+  const [result, setResult] = useState(0);
+
+  const handleClickPanel1 = (num) => {
+    updateNum1(num);
+  };
+
+  const handleClickPanel2 = (num) => {
+    updateNum2(num)
+  };
+
+  const handleClickOperator = (operator) => {
```

Comment:
i would rename this to `updateOperator` or something like that as it's more clear

Reply:
Done!